core/src/test/java/bio/terra/pearl/core/service/export/EnrolleeImportServiceTests.java

```diff
@@ -73,23 +74,73 @@ public class EnrolleeImportServiceTests extends BaseSpringBootTest {
     @Test
     @Transactional
     public void testImportEnrolleesCSV(TestInfo info) {
+        //tests new import and update import
```

Comment:
combined new import and then update test cases too. 
Maybe they can be separated but didnt wanted to reuse already populated test portal/study env

Reply:
not sure I agree -- seems like this test is now too long.  Could you pull the first part of the method into a helper method "setupImportTest()"   And then have one test that tests a single import, and then an separate test that tests updates?

Reply:
+1 Devon, I don't think these should be combined- it feels like this is pushing past the limits of what could be considered a "unit" test